### DISCUSSION TITLE
Surface permissions.limit in the app

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#5108] [Feature Requests]
Allows you to set a maximum number of items the user can read/edit/delete at a time.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
